packages/common/test/ArcGISContextManager.test.ts

```diff
@@ -118,6 +118,28 @@ describe("ArcGISContext:", () => {
       // RequestOptions
       expect(mgr.context.requestOptions.portal).toBe(mgr.context.sharingApiUrl);
       expect(mgr.context.requestOptions.authentication).not.toBeDefined();
+      expect(mgr.context.properties).not.toBeDefined();
+      // now call setProperties and ensure it's returned on context
+      const site = {
+        id: "bc3",
+        groups: ["3ef", "00c"],
+      };
+      mgr.setProperties({ site });
```

Comment:
FYI - I'm thinking in the hub app we'll use `hubSite` instead of `site` since the context classes are meant to be a platform thing and not be specific to the Hub.